packages/grid/src/ExpandableGridModel.ts

```diff
@@ -30,8 +33,23 @@ export interface ExpandableGridModel extends GridModel {
    * Change the expanded status of an expandable row
    * @param row Row to expand
    * @param isExpanded True to expand the row, false to collapse
+   * @param expandDescendants True to expand nested rows, false otherwise
    */
-  setRowExpanded(row: ModelIndex, isExpanded: boolean): void;
+  setRowExpanded(
+    row: ModelIndex,
+    isExpanded: boolean,
+    expandDescendants?: boolean
+  ): void;
+
+  /**
+   * Expand all rows
+   */
+  setExpandAll(): void;
+
+  /**
+   * Collapse all rows
+   */
+  setCollapseAll(): void;
```

Comment:
I just realized - we're going to need to check if `expandAll` functionality is even available, otherwise this change is going to break tree tables on Enterprise.
Specifically, we'll need to add an optional `isExpandAllAvailable?: boolean` flag on `ExpandableGridModel`, then in `IrisGridTreeTableModel`, can implement it like 
```
get isExpandAllAvailable(): boolean {
  return this.table.expandAll !== undefined;
}
```

Then in the menu items, only display Expand/Collapse all for models that support it; in `IrisGridTreeTableModel` don't provide a third param to `table.setExpanded` if `expandAll` isn't available (since that param won't be valid on Enterprise).

I'll need to test this out with Enterprise to make sure it doesn't break existing tree table functionality.

Reply:
When I do ```model.isExpandAllAvailable```, it doesn't run the function but instead accesses the field in ExpandableGridModel which is always undefined. How can I access this function instead?

Reply:
Did you wire it through `IrisGridProxyModel` as well?